running/SELinuxEscape/README.md

```diff
@@ -0,0 +1,6 @@
+# SELinuxEscape
+
+This script demonstrates what would happen if a container was to escape confinement
```

Comment:
nit: missing dot at the end of the sentence.

Reply:
Fixed